projects/cashmere/src/lib/table/bind-observable/bind-observable-options.ts

```diff
@@ -0,0 +1,17 @@
+export interface BindObservableOpts {
+    /**
+     * optional custom key of the companion property.
+     * If not provided, the decorator will take the name of the original property key with a '$' suffix.
+     */
+    key?: string;
+    /**
+     * If true, the observable will emit the raw setter value if the decorated property is an accessor.
+     * Otherwise the observable will call the getter once and emit the returned value.
+     * This option will be ignored if the decorated property has no get accessor.
+     */
+    emitRawSetterValue?: boolean;
+}
+
+export function isBindObservableOpts(pet: any): pet is BindObservableOpts {
```

Comment:
Just curious what lead you to choose `pet` as the variable name. Seems kind of funky haha!

Reply:
Hah - afraid I can't claim to know much of anything when it comes to the bind observable functions, they're an open source utility set that the person who originally posted their column resizing solution was using:

https://github.com/PSanetra/bind-observable

So who knows what they were thinking when they named their variables :-)

Reply:
Got it! I didn't see any source attribution so I assumed it was something you wrote. Think we should include that there?

Reply:
Yup - sounds good.  I put the copyright and license at the top of the file, but I'll attribute it as well.